massgov/mayflower-patternlab
# Fixes Dp 1187 rte extra paragraph

Issue: Extra paragraph tag output on rendering of RTE paragraphs
https://jira.state.ma.us/browse/DP-1187

To test:
- Navigate to `http://mass.local/apply-nursing-license-new-graduate`
- Inspect 'Register for the NCLEX'
- confirm there is only one paragraph tag

Note: This twig template is called in several places:
![image](https://cloud.githubusercontent.com/assets/3721535/22249709/0df48866-e212-11e6-895c-9ebf37e04ad1.png)
As well as dynamically loaded in places like 
![image](https://cloud.githubusercontent.com/assets/3721535/22249761/4402c878-e212-11e6-8789-6b87644aa746.png) 
which is rendered as the following
![image](https://cloud.githubusercontent.com/assets/3721535/22249859/ba3bfc3a-e212-11e6-83a2-b5d196d7c2b2.png)
